readme.md

```diff
@@ -1,15 +1,15 @@
 # Product cards
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_product-cards/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_product-cards/report/html_report/)
+- [DEMO LINK](https://maryanmats.github.io/layout_cards/)
+- [TEST REPORT LINK](https://maryanmats.github.io/layout_cards/report/html_report/)
```

Comment:
I don't understand why you renamed layout_product-cards to layout_cards.

Reply:
@olehmahera Maybe it had changed programmatically. product-card in HTML had changed to just card using command(ctrl) shift f

Reply:
> @olehmahera Maybe it had changed programmatically. product-card in HTML had changed to just card using command(ctrl) shift f

yeah, it's just ctrl+shift+f